deps/chakrashim/src/inspector/v8-debugger.cc

```diff
@@ -77,22 +77,25 @@ void V8Debugger::getCompiledScripts(
     int contextGroupId,
     std::vector<std::unique_ptr<V8DebuggerScript>>& result) {
   JsValueRef scripts;
-  JsDiagGetScripts(&scripts);
+  if (JsDiagGetScripts(&scripts) != JsNoError) {
+    assert(false);
```

Comment:
Looking at code only expected failure is JsErrorDiagNotInDebugMode, in all other case you should at least get an empty array, so assert in all other cases.